src/Libraries/Microsoft.Extensions.Http.Telemetry/Logging/Internal/LogRecordPooledObjectPolicy.cs

```diff
@@ -16,8 +16,8 @@ public override bool Return(LogRecord record)
         record.Path = string.Empty;
         record.Duration = 0;
         record.StatusCode = null;
-        record.RequestBody = string.Empty;
-        record.ResponseBody = string.Empty;
+        record.RequestBody = null;
```

Comment:
How about getting rid of this LogRecordPooledObjectPolicy object and instead having LogRecord implement IResettable? Less code overall.

Reply:
Done